apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -72,7 +72,10 @@ public abstract partial class BigtableClient
             var channelOptions = new[]
             {
                 new ChannelOption(ChannelOptions.MaxSendMessageLength, -1),
-                new ChannelOption(ChannelOptions.MaxReceiveMessageLength, -1)
+                new ChannelOption(ChannelOptions.MaxReceiveMessageLength, -1),
+
+                // Use a random arg to prevent sub-channel re-use in gRPC, which would pin all channels to a single GFE CPU.
+                new ChannelOption("sub-channel-separator", Guid.NewGuid().ToString())
```

Comment:
Ok, I finally figured out how to verify this using an undocumented grpc tracer, and surprisingly the problem _didn't_ exist before, even though it seems like it should have based on everything we've learned. The fixes don't cause any harm, so it should be safe to leave in to make sure things keep working if/when things start working as we assume they would, but I don't feel too strongly about it. As long as I can figure out how to add tests based on this tracer, I'll be happy without having these dummy options. I'll try to look into feasibility of testing soon.
cc @jskeet

Reply:
Thanks Mike - it would definitely be good to have tests for this. If it stops working with a later edition of gRPC, that raises the spectre of a user with a current version of Spanner (etc) updating to a later version of gRPC without updating their Spanner/whatever library and getting caught out, which isn't ideal. Hmm...

Reply:
Right, so it may be good to keep them. I'll leave them in for now and comment back on the internal thread about this to see if we can get some more info.